db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
(this is a question about our intern conventions, not about the code itself. This code should work with the new and the old version, I tested both on my system)
This migration was written when "event" was still called "workshop". Personally I think it should stay workshop in this method, otherwise we could rename all other migration methods and remove the "workshop->event" migration (This would however probably mean that everyone has to reset their database at least once). Do we have conventions/rules for this?